guides/common/assembly_administering-hosts.adoc

```diff
@@ -22,6 +22,26 @@ endif::[]
 
 include::modules/proc_changing-the-environment-of-a-host.adoc[leveloffset=+1]
 
+include::modules/con_managing-multiple-content-view-environments.adoc[leveloffset=+1]
+
+include::modules/con_content-view-environment-ordering-and-priority.adoc[leveloffset=+1]
+
+include::modules/con_comparing-multiple-content-view-environments-and-composite-content-views.adoc[leveloffset=+1]
+
+include::modules/con_handling-content-view-environments-during-registration-with-multiple-activation-keys.adoc[leveloffset=+1]
```

Comment:
Consider moving the AK modules to the AK section of the Managing Content guide. You might want to create an assembly just for multiple content views and activation keys so that they do not all appear at the top level of that section.